evm_trace/gas.py

```diff
@@ -1,54 +1,81 @@
-from typing import Dict, List
+from typing import Any, Dict, List
+
+from hexbytes import HexBytes
+from pydantic import BaseModel, ValidationError, validator
 
 from evm_trace.base import CallTreeNode
+from evm_trace.utils import _convert_hexbytes
+
+Report = Dict[str, Dict[str, List[int]]]
```

Comment:
I like `GasReport`; it gives the class more context without having to see where it is coming from.
`Report` sounds like a base class or an abstract class

Reply:
Moved back to GasReport [a72b3f1](https://github.com/ApeWorX/evm-trace/pull/14/commits/a72b3f12de70f0c1ec75836fa6b8a3f422f29d56)